site/content/docs/5.2/forms/overview.md

```diff
@@ -160,6 +160,67 @@ Use the `.is-required` class on `label` to add a `*` symbol after it.
 </form>
 {{< /example >}}
 
+## Form helper
+
+Provide some help on an input with `.btn-helper`. **Be sure to include text for screen readers**, as we've done with `<span class="visually-hidden">`.
+
+On top on this button, apply a [popover]({{< docsref "/components/popovers" >}}) to display the help. A [tooltip]({{< docsref "/components/tooltips" >}}) can also be applied for small and simple text helpers without any link, button, etc...
+
+{{< example >}}
+<button type="button" class="btn-helper" data-bs-toggle="popover" data-bs-placement="top" data-bs-content="Help with popover that could be very long with various content.">
+  <span class="visually-hidden">Helper for input</span>
+</button>
+<button type="button" class="btn-helper" data-bs-toggle="tooltip" data-bs-placement="top" data-bs-title="Help with tooltip.">
+  <span class="visually-hidden">Helper for input</span>
+</button>
+{{< /example >}}
+
+The helper button should be displayed right after a label.
+
+{{< example >}}
+<div class="mb-3">
+  <label for="tooltipTextInput" class="form-label">Tooltip input</label>
+  <button type="button" class="btn-helper" data-bs-toggle="popover" data-bs-placement="top" data-bs-content="Help for input">
+    <span class="visually-hidden">Helper for input</span>
+  </button>
+  <input type="text" id="tooltipTextInput" class="form-control" placeholder="Tooltip input">
+</div>
+<div>
+  <label for="tooltipSelect" class="form-label is-required">Tooltip select menu</label>
+  <button type="button" class="btn-helper" data-bs-toggle="tooltip" data-bs-placement="top" data-bs-title="Help for select menu">
+    <span class="visually-hidden">Helper for select menu</span>
+  </button>
+  <select id="tooltipSelect" class="form-select" required>
+    <option value="">Tooltip select</option>
+  </select>
+</div>
+{{< /example >}}
+
+<!-- To remove just here to show that it could be well and easily integrated everywhere -->
```

Comment:
You can remove this part and include it in the description for the future so we don't forget it